HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentViewModel.swift

```diff
@@ -29,54 +30,94 @@ final class PaymentComponentViewModel {
+            return isPaymentProviderInstalled ? bankName : placeholderBankNameText
+        }
+        return placeholderBankNameText
+    }
     var bankNameLabelFont: UIFont
     let bankNameLabelAccentColor: UIColor = GiniColor(lightModeColor: UIColor.GiniColors.dark1, 
                                                       darkModeColor: UIColor.GiniColors.light1).uiColor()
+    private let placeholderBankNameText: String = NSLocalizedStringPreferredFormat("ginihealth.paymentcomponent.selectBank.label", comment: "")
     let chevronDownIconName: String = "iconChevronDown"
     
-    // pay invoice view
-    let payInvoiceViewBackgroundColor: UIColor
+    // pay invoice view background color
+    var payInvoiceViewBackgroundColor: UIColor {
+        if let payInvoiceViewBackgroundColorString, let backgroundHexColor = payInvoiceViewBackgroundColorString.toColor() {
+            return isPaymentProviderInstalled ? backgroundHexColor : defaultPayInvoiceViewBackgroundColor.withAlphaComponent(0.4)
+        }
+        return defaultPayInvoiceViewBackgroundColor.withAlphaComponent(0.4)
+    }
+    private let payInvoiceViewBackgroundColorString: String?
+    private let defaultPayInvoiceViewBackgroundColor: UIColor = UIColor.GiniColors.accent1
+
+    // pay invoice view text color
     let payInvoiceLabelText: String = NSLocalizedStringPreferredFormat("ginihealth.paymentcomponent.payInvoice.label", comment: "")
-    let payInvoiceLabelAccentColor: UIColor
+    var payInvoiceLabelAccentColor: UIColor {
+        if let payInvoiceLabelAccentColorString, let textHexColor = payInvoiceLabelAccentColorString.toColor() {
+            return textHexColor
+        }
+        return defaultpayInvoiceLabelAccentColorString
+    }
+    private let payInvoiceLabelAccentColorString: String?
+    private let defaultpayInvoiceLabelAccentColorString: UIColor = UIColor.GiniColors.dark7
+
+    // pay invoice view font
     let payInvoiceLabelFont: UIFont
-    
+
+    var isPaymentProviderInstalled: Bool {
+        if let paymentProviderScheme, let url = URL(string: paymentProviderScheme), UIApplication.shared.canOpenURL(url) {
+            return true
+        }
+        return false
+    }
+    private var paymentProviderScheme: String?
+
     weak var delegate: PaymentComponentViewModelProtocol?
     
-    init(bankName: String,
-         bankIconName: String, 
-         payInvoiceAccentColor: GiniColor,
-         payInvoiceTextColor: GiniColor,
+    init(paymentProvider: PaymentProvider?,
          giniHealth: GiniHealth) {
         self.giniHealth = giniHealth
         self.moreInformationLabelFont = GiniHealthConfiguration.shared.customFont.with(weight: .regular,
-                                                                          size: 13,
-                                                                          style: .caption1)
+                                                                                       size: 13,
```

Comment:
[Clarification]: why we don't use here GiniHealthConfiguration.shared.textStyleFonts[.caption1] instead?